README.md

```diff
@@ -54,7 +54,7 @@ All HNC issues are assigned to an HNC milestone. So far, the following
 milestones are defined or planned:
 
 * v1.0 - Likely late Q2 or early Q3 2021: HNC recommended for production use
-* [v0.9](https://github.com/kubernetes-sigs/multi-tenancy/milestone/21): move
+* [v0.9](https://github.com/kubernetes-sigs/hierarchical-namespaces/milestone/21): move
```

Comment:
Maybe create these milestones in the new repo so we can have the correct number here? But you can also do that in a followup PR if you like.

Reply:
I think the milestones will be carried over when we use the tool to copy the issues. @yiqigao217 can you confirm? I'd prefer to use that than do them manually now and have it mess up the migration